File: beginner/javascript/1983.js
let input = require('fs').readFileSync('stdin', 'utf-8').split('\n');

var nota, maior = 0;
var n, i, matricula, matricular_maior = 0;

n = parseInt(input.shift());

for (i = 0; i < n; i++) {
    var valores = input.shift().split(" ");
    matricula = parseInt(valores[0]);
    nota = parseFloat(valores[1]);

    if (nota > maior) {
        maior = nota;
        matricular_maior = matricula;
    }
}

if (maior >= 8) {
    console.log(matricular_maior);
} else {
    console.log("Minimum note not reached");
}

File: beginner/python/1983.py
maior = 0
matricular_maior = 0

n = int(input())

for i in range(0,n):
    valores = input().split()
    matricula = int(valores[0])
    nota = float(valores[1])

    if (nota > maior):
        maior = nota
        matricular_maior = matricula

if (maior >= 8):
    print(matricular_maior)
else:
    print("Minimum note not reached")
